src/lambda/expression.rs

```diff
@@ -22,6 +22,7 @@ pub enum Expression {
     Cache(Cache),
     Path(Box<Expression>, Vec<String>),
     Protect(Box<Expression>),
+    Js(Arc<dyn WorkerIO<Option<ConstValue>, ConstValue>>),
```

Comment:
Worker should not be stored in the Expression. It should be part of the runtime.